sensor_msgs/Image im
dlibRect[] rects
---
faceDetails[] person
